js/ngermrest.js

```diff
@@ -3,4 +3,46 @@ angular.module('ERMrest', [])
     .factory('ermrestServerFactory', ['$http', '$q', function($http, $q) {
         ERMrest.configure($http, $q);
         return ERMrest.ermrestFactory;
-    }]);
\ No newline at end of file
+    }])
+    .factory('myHttpInterceptor', ['$q', function($q){
```

Comment:
This probably doesn't belong in the _ngermrest.js_ script not as a new angular service that we are providing to all our clients. Or does Chaise now depend on it for some reason?